minecraft_servers/dragonpvp/manifest.json

```diff
@@ -8,7 +8,7 @@
     "social": {
       "web": "https://www.dragonpvp.site",
       "web_shop": "https://www.dragonpvp.site/store",
-      "web_support": "https://www.dragonpvp.site/forum",
+      "web_support": "https://www.dragonpvp.site/supporto",

```

Comment:
Also cannot be just text "join Discord"; remove if something like a FAQ is non-existent